### DISCUSSION TITLE
Please add the "lib/anime.es.d.ts" file to [@types/animejs]

### DISCUSSION THREAD

**Discussion by wisdomstar94** [DefinitelyTyped/DefinitelyTyped#62061] [Issues with a @types package]
`import anime from 'animejs/lib/anime.es';`

If you import animejs/lib/anime.es as above, Even if @types/animejs is installed, the type is not specified.
To work around this, copy the existing @types/animejs/index.d.ts file to the @types/animejs/lib/anime.es.d.ts file.
Please modify the package so that the lib/anime.es.d.ts file exists when @types/animejs is installed.

<img width="966" alt="image" src="https://user-images.githubusercontent.com/93423564/187801312-7bec6182-28b0-43b0-8175-91c4854179c8.png">

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
I just opened #62537
